skfp/preprocessing/input_output/aminoseq.py

```diff
@@ -0,0 +1,93 @@
+from collections.abc import Sequence
+from typing import Optional
+
+from rdkit.Chem import Mol, MolFromFASTA
+
+from skfp.bases import BasePreprocessor
+from skfp.utils.validators import check_strings
+
+
+class MolFromAminoseqTransformer(BasePreprocessor):
+    """
+    Creates RDKit ``Mol`` objects from amino-acid sequence strings.
+    This implementation uses sequences defined by FASTA [1]_ format.
+
+    Parameters
+    ----------
+    sanitize : bool, default=True
+        Whether to perform sanitization, i.e. basic validity checks, on created
+        molecules. For details see RDKit documentation [2]_.
+
+    flavor : int, default=1
+        Type of molecule [3]_.
+
+
+    n_jobs : int, default=None
+        The number of jobs to run in parallel. :meth:`transform` is parallelized
+        over the input molecules. ``None`` means 1 unless in a
+        :obj:`joblib.parallel_backend` context. ``-1`` means using all processors.
+        See Scikit-learn documentation on ``n_jobs`` for more details.
+
+    batch_size : int, default=None
+        Number of inputs processed in each batch. ``None`` divides input data into
+        equal-sized parts, as many as ``n_jobs``.
+
+    verbose : int, default=0
+        Controls the verbosity when processing molecules.
+
+    References
+    ----------
+    .. [1] `Lipman DJ, Pearson WR.
+        "Rapid and sensitive protein similarity searches."
+        Science. 1985 Mar 22; 227(4693):1435-41.
+        <https://pubmed.ncbi.nlm.nih.gov/2983426/>`_
+
+    .. [2] `Gregory Landrum
+        "The RDKit Book: Molecular Sanitization"
+        <https://www.rdkit.org/docs/RDKit_Book.html#molecular-sanitization>`_
+
+    .. [3] `RDKit documentation
```

Comment:
Be slightly more specific, e.g. "RDKit documentation - MolFromFASTA" or something